lib/cocoapods/installer.rb

```diff
@@ -155,6 +155,7 @@ def resolve_dependencies
         validate_build_configurations
         clean_sandbox
       end
+      analyzer
```

Comment:
Returning this here solely to make it easier to test. I'm open to other ideas on a better way to get access to the analyzer